### PR TITLE
Add AirGradient button docs

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add AirGradient button docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/119917
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Enhanced `airgradient` integration with support for requesting CO2 calibration.
  - Added functionality to request LED bar test in `airgradient` integration.
  - Introduced new platforms: `button` and `select` for `airgradient` integration.

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/airgradient.markdown [hunk 1]
```diff
@@ -57,3 +59,5 @@ The settings are only applied to the device when the configuration source is set
 The following entities are supported:
 
 - Display temperature unit
+- Requesting co2 calibration
+- Requesting led bar test
```
Comment (c0ffeeca7) [inline source/_integrations/airgradient.markdown]:
```suggestion
- Requesting LED bar test
```

File: source/_integrations/airgradient.markdown [hunk 2]
```diff
@@ -57,3 +59,5 @@ The settings are only applied to the device when the configuration source is set
 The following entities are supported:
 
 - Display temperature unit
+- Requesting co2 calibration
```
Comment (c0ffeeca7) [inline source/_integrations/airgradient.markdown]:
```suggestion
- Requesting CO2 calibration
```

**Review by c0ffeeca7 — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @joostlek 👍 

../Frenck

**Pull request merged**